Content.Server/Mining/Electromagnet.cs

```diff
@@ -0,0 +1,79 @@
+using Content.Server.Power.Components;
+using Content.Shared.Tag;
+using Robust.Shared.Physics.Components;
+using Robust.Shared.Physics.Systems;
+
+namespace Content.Server.Mining;
+
+[RegisterComponent]
+public class ElectromagnetComponent : Component
+{
+    [DataField("range")]
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float Range = 2f;
+
+    [DataField("force")]
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float Force = 300f;
+
+    [DataField("minDist")]
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float MinDist = 0.5f;
+}
+
+public class ElectromagnetSystem : EntitySystem
+{
+    [Dependency] private readonly EntityLookupSystem _lookup = default!;
+    [Dependency] private readonly TagSystem _tagSystem = default!;
+    [Dependency] private readonly SharedPhysicsSystem _physics = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+    }
```

Comment:
This is not a draft, there's just nothing to initialize. I did this because a lot of other places in the code call `base.Initialize()` despite it not really doing anything. I can remove this empty Initialize() method if that is preferred.

Reply:
Nowhere else calls an empty initialize...

Reply:
A global grep of all calls to EntitySystem's `base.Initialize()` begs to differ. Again, I can just remove this if this is the preference.

Reply:
They call base.Initialize and actually do things in their own Initialize method, this has nothing to be overridden.